packages/@okta/vuepress-theme-prose/components/Resources.vue

```diff
@@ -0,0 +1,154 @@
+                  fill="currentColor"
+                  mask="url(#path-1-inside-1_1471_16106)"
+                />
+              </svg>
+            </a>
+          </div>
+        </li>
+        <li class="resources__list-item">
+          <h3 class="resources__list-title dont-break-out">
+            Technical documentation
+          </h3>
+          <p class="resources__list-text dont-break-out">
+            Discover official Terraform partner resources to automate provisioning and management for Workforce Identity.
+          </p>
+          <div class="resources__links">
+            <a
+              class="resources__list-link dont-break-out"
+              href="https://support.okta.com/help/s/article/Prepare-the-Okta-Terraform-Provider-for-the-OIE-upgrade?language=en_US"
+            >
+              <span>Prepare the Okta Terraform Provider for OIE Upgrade</span>
+              <svg
+                width="13"
+                height="13"
+                viewBox="0 0 13 13"
+                fill="none"
+                xmlns="http://www.w3.org/2000/svg"
+                aria-hidden="true"
+                focusable="false"
+              >
+                <path
+                  d="M9.552 4.064L9.536 7.968H11.056V1.472H4.56V2.992L8.448 2.976L4 7.408L5.12 8.528L9.552 4.064Z"
+                  fill="currentColor"
+                />
+                <path
+                  d="M0 0.472H12.056H0ZM12.056 14.028H2C0.0670034 14.028 -1.5 12.461 -1.5 10.528H1.5C1.5 10.8041 1.72386 11.028 2 11.028H12.056V14.028ZM2 14.028C0.0670034 14.028 -1.5 12.461 -1.5 10.528V0.472H1.5V10.528C1.5 10.8041 1.72386 11.028 2 11.028V14.028ZM12.056 0.472V12.528V0.472Z"
+                  fill="currentColor"
+                  mask="url(#path-1-inside-1_1471_16106)"
+                />
+              </svg>
+            </a>
+            <a
+              class="resources__list-link dont-break-out"
+              href="https://github.com/okta/terraform-provider-okta"
+            >
+              <span>View Terraform provider on Github</span>
+              <svg
+                width="13"
+                height="13"
+                viewBox="0 0 13 13"
+                fill="none"
+                xmlns="http://www.w3.org/2000/svg"
+                aria-hidden="true"
+                focusable="false"
+              >
+                <path
+                  d="M9.552 4.064L9.536 7.968H11.056V1.472H4.56V2.992L8.448 2.976L4 7.408L5.12 8.528L9.552 4.064Z"
+                  fill="currentColor"
+                />
+                <path
+                  d="M0 0.472H12.056H0ZM12.056 14.028H2C0.0670034 14.028 -1.5 12.461 -1.5 10.528H1.5C1.5 10.8041 1.72386 11.028 2 11.028H12.056V14.028ZM2 14.028C0.0670034 14.028 -1.5 12.461 -1.5 10.528V0.472H1.5V10.528C1.5 10.8041 1.72386 11.028 2 11.028V14.028ZM12.056 0.472V12.528V0.472Z"
+                  fill="currentColor"
+                  mask="url(#path-1-inside-1_1471_16106)"
```

Comment:
All of these SVGs in this file appear to be the same image. Let's move it to its own file so we don't have to duplicate the inline SVG multiple times.